pkgs/games/freeciv/default.nix

```diff
@@ -1,32 +1,38 @@
 { stdenv, fetchurl, zlib, bzip2, pkgconfig, curl, lzma, gettext
-, sdlClient ? true, SDL, SDL_mixer, SDL_image, SDL_ttf, SDL_gfx, freetype
+, sdlClient ? true, SDL, SDL_mixer, SDL_image, SDL_ttf, SDL_gfx, freetype, fluidsynth
 , gtkClient ? false, gtk
 , server ? true, readline }:
 
 let
   inherit (stdenv.lib) optional optionals;
-  client = sdlClient || gtkClient;
 
   sdlName = if sdlClient then "-sdl" else "";
   gtkName = if gtkClient then "-gtk" else "";
 
-  baseName = "freeciv-2.4.0";
+  baseName = "freeciv-2.5.0";
 in
 stdenv.mkDerivation {
   name = baseName + sdlName + gtkName;
```

Comment:
Putting texts *after* version is ugly, I believe. It doesn't work well with our version/update semantics.